tasks/test-project/tasks.js

```diff
@@ -156,7 +156,7 @@ async function webTasks(outputPath, { link, verbose }) {
         // @NOTE: use rwfw, because calling the copy function doesn't seem to work here
         task: () =>
           execa(
-            'yarn workspace web add postcss postcss-loader tailwindcss autoprefixer',
+            'yarn workspace web add postcss postcss-loader tailwindcss autoprefixer@^9.8.8',
```

Comment:
If this turns out to be necessary, I wonder if we can handle it via storybook/main.js config similar to the way Chakra UI had to over here:
https://github.com/redwoodjs/redwood/blob/e87f741906d9ff0ab9b8801795493e68939bb6c4/packages/testing/config/storybook/main.js#L27-L37

Reply:
In my testing this is necessary.

For handling the config, not sure if I fully understood how to implement, but I attempted something here: 8522cbf6938ffd56866ef3e4ddb42eb4a99fd3b3

Reply:
Yeah, something to that end is the idea. Did you have any success with https://github.com/redwoodjs/redwood/commit/8522cbf6938ffd56866ef3e4ddb42eb4a99fd3b3 ?

Reply:
Actually, hold up a minute... this might not be related to Storybook at all. I just tried to reproduce and couldn't get tailwind working on the most recent Redwood v0.41 🤦‍♂️

See https://github.com/redwoodjs/redwood/issues/4144